src/util/navigation.js

```diff
@@ -190,13 +188,6 @@ export function removeGroupFromUrl (url) {
   return url.replace(new RegExp(matchForReplaceRegex), '')
 }
 
-export function getGroupSlugInPath (pathname) {
```

Comment:
Wasn't being used, and from a `react-router` / routing perspective methods such as this are smelly.